frontend/src/component/project/Project/ProjectSettings/ProjectDefaultStrategySettings/ProjectEnvironment/ProjectEnvironmentDefaultStrategy/ProjectEnvironmentDefaultStrategy.tsx

```diff
@@ -51,44 +48,42 @@ const ProjectEnvironmentDefaultStrategy = ({
         environmentId,
     );
 
-    const strategy: CreateFeatureStrategySchema = useMemo(() => {
-        return defaultStrategy ? defaultStrategy : DEFAULT_STRATEGY;
+    const strategy: Omit<IFeatureStrategy, 'id'> = useMemo(() => {
+        const baseDefaultStrategy = defaultStrategy
+            ? defaultStrategy
+            : DEFAULT_STRATEGY;
+        return {
+            ...baseDefaultStrategy,
+            disabled: false,
+            constraints: baseDefaultStrategy.constraints ?? [],
+            title: baseDefaultStrategy.title ?? '',
+            parameters: baseDefaultStrategy.parameters ?? {},
+        };
     }, [JSON.stringify(defaultStrategy)]);
 
     return (
-        <>
-            <StrategyItemContainer
-                strategy={strategy as any}
-                description={description}
-                actions={
-                    <>
-                        <PermissionIconButton
-                            permission={[
-                                PROJECT_DEFAULT_STRATEGY_WRITE,
-                                UPDATE_PROJECT,
-                            ]}
-                            environmentId={environmentId}
-                            projectId={projectId}
-                            component={Link}
-                            to={editStrategyPath}
-                            tooltipProps={{
-                                title: `Edit default strategy for "${environmentId}"`,
-                            }}
-                            data-testid={`STRATEGY_EDIT-${strategy?.name}`}
-                        >
-                            <Edit />
-                        </PermissionIconButton>
-                    </>
-                }
-            >
-                <StrategyExecution strategy={strategy} />
-
-                {strategy.variants && strategy.variants.length > 0 ? (
-                    <SplitPreviewSlider variants={strategy.variants} />
-                ) : null}
-            </StrategyItemContainer>
-        </>
+        <StrategyItem
+            strategy={strategy}
+            headerItemsRight={
```

Comment:
Maybe "actions"/"headerActions" prop is better, because it's less coupled to the visual design?

Reply:
Yeah, maybe. The thing is that when we get to the playground, there's also left-aligned items. We can either solve that by the caller passing in a container that wraps it or by adding right and left props. But I wanna delay this decision until we've found all the places we use it and then take it from there.